stock_operating_unit/model/stock_move_line.py

```diff
@@ -0,0 +1,61 @@
+# © 2020 brain-tec group
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    operating_unit_id = fields.Many2one(
+        "operating.unit",
+        compute="_compute_operating_unit_id",
+        store=True,
+        string="Source Location Operating Unit",
+    )
+
+    operating_unit_dest_id = fields.Many2one(
+        "operating.unit",
+        compute="_compute_operating_unit_dest_id",
+        store=True,
+        string="Dest. Location Operating Unit",
+    )
+
+    @api.depends(
+        "location_id",
+        "location_id.operating_unit_id",
+        "picking_id.picking_type_id",
+        "picking_id.picking_type_id.warehouse_id",
+        "picking_id.picking_type_id.warehouse_id.operating_unit_id",
```

Comment:
I think it would be enough if you only add "location_id.operating_unit_id" and "picking_id.picking_type_id.warehouse_id.operating_unit_id" as dependency. The other 3 fields you can remove.
As example the "location_id" field is not necessary because imagine you have two locations with the same operating unit. If you change the location of the stock.move.line between this two locations the locations changes but the operating_unit of the rec.location_id.operating_unit_id stays the same. Therefore the compute method don't need to be always called. Only if the operating unit of the location changes.
If you only add that 2 fields you can save some performance.
You don't need to change this but keep it in mind for future.